### PR TITLE
[NR-218026] Add Unreal as ApplicationFramework

### PR DESCRIPTION
**Pull request by ndesai-newrelic**

### CODE REVIEW

File: agent-core/src/main/java/com/newrelic/agent/android/ApplicationFramework.java [hunk 1]
```diff
@@ -14,5 +14,8 @@ public enum ApplicationFramework {
     Xamarin,
     MAUI,
```
Comment (ywang-nr) [inline agent-core/src/main/java/com/newrelic/agent/android/ApplicationFramework.java:16]: Do we want to remove the empty lines?

File: agent-core/src/main/java/com/newrelic/agent/android/ApplicationFramework.java [hunk 2]
```diff
@@ -14,5 +14,8 @@ public enum ApplicationFramework {
...
 
-    Unity
+    Unity,
+
```
Comment (ywang-nr) [inline agent-core/src/main/java/com/newrelic/agent/android/ApplicationFramework.java:18]: Do we want to remove the empty lines?

Comment (ndesai-newrelic) [reply agent-core/src/main/java/com/newrelic/agent/android/ApplicationFramework.java]: done.

**Review by ywang-nr — APPROVED** [review]
Looks good

**Pull request merged**